### PR TITLE
Fix contact link for CommCareCaseSQL recipients in messaging reports

### PR DESCRIPTION
**Pull request by gcapalbo**
http://manage.dimagi.com/default.asp?247802

@orangejenny

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Code looks great. It doesn't look like tests kicked off. Glitch in travis?

**Comment by gcapalbo** [comment]
yeah, looks like it, i'll merge master

File: corehq/apps/reports/standard/sms.py [hunk 1]
```diff
@@ -397,7 +393,7 @@ def get_timestamp(date_):
             return table_cell['html']
 
         def get_contact_link(couch_recipient, couch_recipient_doc_type, raw=False):
-            doc_info = self.get_recipient_info(couch_recipient_doc_type, couch_recipient, contact_cache)
+            doc_info = self.get_recipient_info(self.domain, couch_recipient_doc_type, couch_recipient, contact_cache)
```
Comment (dimagimon) [inline corehq/apps/reports/standard/sms.py]: E501 line too long (117 > 115 characters)

**Comment by gcapalbo** [comment]
approved, build fixed, merging

**Pull request merged**